Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -22,6 +22,13 @@ def runSelected(opt):
     mrd = MatrixReader(opt)
     mrd.prepare(opt.useInput, opt.refRel, opt.fromScratch)
 
+    # test for wrong input workflows
+    if opt.testList:
+        definedWF = []
```

Comment:
it might be worth reimplementing this using set operations (faster, and also has the benefit of identifying all undefined workflows rather than just stopping at the first):
```python
definedSet = set([dwf.numId for dwf in mrd.workFlows])
testSet = set(opt.testList)
undefSet = definedSet - testSet
if len(undefSet)>0: raise ValueError('Undefined workflows: '+str(undefSet))
```